simplex/bls_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package simplex
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+)
+
+func TestBLSVerifier(t *testing.T) {
+	config, err := newEngineConfig()
+	require.NoError(t, err)
+	signer, verifier := NewBLSAuth(config)
+	otherNodeID := ids.GenerateTestNodeID()
+
+	msg := []byte("Begin at the beginning, and go on till you come to the end: then stop")
+	tests := []struct {
+		name      string
+		expectErr error
+		nodeID    []byte
+		sig       []byte
+	}{
+		{
+			name:      "valid signature",
```

Comment:
I have recently been convinced that test names should use `_` rather than spaces so that the golang test names displayed during test failures textually match the code for easier searching